--- cvmfs-2.4.4.orig/bootstrap.sh	2017-12-11 14:35:17.000000000 +0100
+++ cvmfs-2.4.4/bootstrap.sh	2018-04-17 12:51:55.231489376 +0200
@@ -244,7 +244,7 @@
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 
 # Build a list of libs that need to be built
-missing_libs="libcurl pacparser zlib sparsehash leveldb googletest libgeoip tbb protobuf googlebench sqlite3 vjson sha2 sha3 mongoose"
+missing_libs="vjson sha2 sha3"
 
 if [ -f $externals_install_dir/.bootstrapDone ]; then
   existing_libs=$(cat $externals_install_dir/.bootstrapDone)
